src/Core/Services/Implementations/PolicyService.cs

```diff
@@ -206,7 +209,9 @@ private async Task<IEnumerable<OrganizationUserPolicyDetails>> QueryOrganization
         }
 
         var excludedUserTypes = GetUserTypesExcludedFromPolicy(policyType);
+        var orgAbilities = await _applicationCacheService.GetOrganizationAbilitiesAsync();
         return _cachedOrganizationUserPolicyDetails.Where(o =>
+            (!orgAbilities.ContainsKey(o.OrganizationId) || orgAbilities[o.OrganizationId].Enabled && orgAbilities[o.OrganizationId].UsePolicies) &&
```

Comment:
Please add brackets to make the order of evaluation clearer:

```suggestion
            (!orgAbilities.ContainsKey(o.OrganizationId) || (orgAbilities[o.OrganizationId].Enabled && orgAbilities[o.OrganizationId].UsePolicies)) &&
```

More importantly - this means that if the org isn't cached in `orgAbilities`, we assume policies do apply? Is there a situation where the org would not be cached?

Reply:
Also, have you confirmed with product that policies do not apply to disabled orgs? It's _probably_ fine, but it is a change in behaviour. For example, if the subscription lapses temporarily and is then reinstated, there's been a window where users could export their personal vault, or login without SSO, or remove 2FA from their account. And these may not be retroactively enforced when the subscription is reinstated. (e.g. users who removed 2FA will remain without 2FA)

Reply:
> More importantly - this means that if the org isn't cached in orgAbilities, we assume policies do apply? Is there a situation where the org would not be cached?

I added this as a precaution in the case we have a connection issue on `ServiceBus` and then the new `OrgAbility` wouldn't propagate throughout the different projects (for example creating an Organization in the Admin project). Does this make sense (or am I overthinking)? 

> Also, have you confirmed with product that policies do not apply to disabled orgs? It's probably fine, but it is a change in behaviour. 

I did not realise I was changing the behaviour, I have reverted this.

Reply:
lgtm, and I think it's sensible to default to the stricter behaviour (assuming policies apply) if the value is missing for any reason.